xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -580,6 +580,7 @@ void CVideoPlayer::CreatePlayers()
   m_VideoPlayerSubtitle = new CVideoPlayerSubtitle(&m_overlayContainer, *m_processInfo);
   m_VideoPlayerTeletext = new CDVDTeletextData(*m_processInfo);
   m_VideoPlayerRadioRDS = new CDVDRadioRDSData(*m_processInfo);
+  m_VideoPlayerAudioID3 = std::make_unique<CVideoPlayerAudioID3>(*m_processInfo);
```

Comment:
OK for now, just need to find a way in the future to create the classes only when used.
The `CDVDTeletextData`, `CDVDRadioRDSData` and `CVideoPlayerAudioID3` relates only to PVR and on normal playbacks possibility never used. Most times then a never used memory.